app/views/orders/_edit_form.html.erb

```diff
@@ -37,15 +53,20 @@
     <%= f.text_field :AdresTo %>
   </div>
   <div class="field">
-    <%= f.label :number_of_passengers %><br>
+    <%= f.label 'Количество пассажиров' %><br>
     <%= f.number_field :number_of_passengers %>
   </div>
   <div class="field">
-    <%= f.label :date_of_trip %><br>
-    <%= f.datetime_select :date_of_trip %>
+    <%= f.label 'Дата и время поездки' %><br>
+    <!--<%= f.datetime_select :date_of_trip %>-->
+    <div class="row">
+      <div class='col-sm-6'>
+          <%= f.text_field :date_of_trip,  :class=>'form-control', :id=>'datetimepicker4' %>
+      </div>
+    </div>
   </div>
   <div class="field">
-    <%= f.label :phone_number %><br>
+    <%= f.label 'Контактный номер' %><br>
```

Comment:
Don't remove `:phone_number`, just add a name as a second argument.
You can read about labels here: http://apidock.com/rails/ActionView/Helpers/FormHelper/label